lib/packwerk/commands/update_todo_command.rb

```diff
@@ -37,14 +37,22 @@ def run
 
         offense_collection = OffenseCollection.new(configuration.root_path)
         offense_collection.add_offenses(offenses)
-        offense_collection.persist_package_todo_files(run_context.package_set)
 
-        out.puts(<<~EOS)
-          #{offenses_formatter.show_offenses(offense_collection.errors)}
-          ✅ `package_todo.yml` has been updated.
-        EOS
+        if offense_collection.strict_mode_violations.any?
```

Comment:
It looks like if there are *any* strict mode violations, we fail out immediately and do not make any changes to TODO files.

What do you think of instead making all the changes we *can* make when updating, but then if there are any strict mode violations, we inform the user and exit with a failure?